react-native-purchases-ui/android/src/main/java/com/revenuecat/purchases/react/ui/FontAssetManager.kt

```diff
@@ -0,0 +1,59 @@
+package com.revenuecat.purchases.react.ui
+
+import android.content.res.AssetManager
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+
+
+object FontAssetManager {
+    private val fontFamilyHashMap: MutableMap<String, FontFamily> = hashMapOf()
```

Comment:
I think this can be a `mutableMap`

Reply:
Also, maybe we are able to make it non mutable by making copies. Not sure if that's worth it, but I am scared of mutable maps, specially if they can be accessed from multiple threads, like in this case.

Reply:
@vegaro oh wow I haven't thought about that, I'll keep it in my mind if I do contribute in the future :)